monkey/infection_monkey/utils/plugins/pluginTests/BadImport.py

```diff
@@ -1,5 +1,3 @@
-from infection_monkey.utils.plugins.pluginTests.PluginTestClass import \
-    TestPlugin  # noqa: F401
 
```

Comment:
Remove, it's there for a reason. Real test

Reply:
Okay. I will remove this.